index.html

```diff
@@ -88,6 +92,132 @@
 				maxime mollitia necessitatibus nobis pariatur quae quaerat quo. Doloremque, dolores eos necessitatibus tempora
 				tenetur vero.
 			</ts-note>
+			<article>
+				<h2>Status</h2>
+				<div class="status-container">
+					<ts-status class="grid-status" status="ok"></ts-status>
+					<ts-status class="grid-status" status="ok" text="custom ok message"></ts-status>
+					<ts-status class="grid-status" status="error"></ts-status>
+					<ts-status class="grid-status" status="error" text="custom error message"></ts-status>
+					<ts-status class="grid-status" status="note"></ts-status>
+					<ts-status class="grid-status" status="note" text="custom note message"></ts-status>
+				</div>
+			</article>
+
+			<article>
+				<h2>Document Card</h2>
+				<div class="document-card-container">
+					<ts-document-card
+						name="Invoice #344234"
+						mobile-description="Tradeshift SRL | 6 participants"
+						description="Tradeshift SRL"
+					></ts-document-card>
+					<ts-document-card
+						name="looooooooooooooooooooooooooooooooooooooooong teeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeext"
+						description="Tradeshift looooooooooooooooooooooooooooooooooooooooong"
+						mobile-description="Mobile looooooooooooooooooooooooooooooooooooooooong"
+					></ts-document-card>
+				</div>
+			</article>
+			<article>
+				<h2>Table</h2>
+				<ts-table id="ts-table-demo"></ts-table>
+				<script>
+					function handleRowClick(e) {
+						console.log('handleRowClick', e);
+					}
+					const tsTable = document.getElementById('ts-table-demo');
+					tsTable.addEventListener('row-click', handleRowClick);
+					tsTable.cols = [
+						{
+							type: 'document',
+							property: 'context',
+							size: 'large',
+							value: 'Context'
+						},
+						{ visibility: 'desktop', type: 'status', property: 'type', size: 'small', value: 'Type', display: 'left' },
+						{ visibility: 'desktop', property: 'participants', size: 'small', value: 'Participants' },
+						{ visibility: 'desktop', property: 'newActivity', value: 'New Activity' },
+						{
+							visibility: 'desktop',
+							property: 'lastActivity',
+							value: 'Last Activity',
+							size: 'medium',
+							display: 'right'
+						},
+						{
+							visibility: 'mobile',
+							property: 'mobileLastActivity',
+							size: 'small',
+							display: 'right',
+							renderFunction(item) {
+								const div = document.createElement('div');
+								div.style.padding = '10px';
+								div.style.whiteSpace = 'nowrap';
+								if (item.unread && item.unread > 0) {
+									const child = document.createElement('div');
+									child.innerText = item.unread;
+									child.style.backgroundColor = '#01b0ff';
+									child.style.borderRadius = '3px';
+									child.style.color = 'white';
+									child.style.padding = '5px';
+									child.style.fontSize = '11px';
+									child.style.fontWeight = '600';
+									div.style.display = 'flex';
+									div.style.alignItems = 'center';
```

Comment:
You can move all these stylings to the index.css to clear the code.

Reply:
cannot do this because of the css encapsulation of shadow dom